### PR TITLE
Add Darter Pro 8

### PR DESCRIPTION
**Pull request by crawfxrd**
system76/firmware-open#322

### CODE REVIEW

File: src/board/system76/darp8/board.mk [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Set touchpad PS2 bus
+CFLAGS+=-DPS2_TOUCHPAD=PS2_3
+
+# Set smart charger parameters
+#TODO: Find out why input current must by divided by two
+CHARGER=oz26786
+CFLAGS+=\
+	-DCHARGER_CHARGE_CURRENT=1536 \
+	-DCHARGER_CHARGE_VOLTAGE=8800 \
+	-DCHARGER_INPUT_CURRENT=2370
```
Comment (crawfxrd) [inline src/board/system76/darp8/board.mk:26]: Should we still be dividing by 2 for ADL?

Comment (jackpot51) [reply src/board/system76/darp8/board.mk:26]: Do you have a killawatt to test with?

Comment (jackpot51) [reply src/board/system76/darp8/board.mk:26]: Charging wattage (system off, battery drained) should be about 30W

Comment (jackpot51) [reply src/board/system76/darp8/board.mk:26]: Charging wattage should drop if the system is on and stress testing caused it to reach the max current to pull from the adapter.

Comment (jackpot51) [reply src/board/system76/darp8/board.mk:26]: System off, measure with killawatt. System on, measure total power with killawatt and charging power with sensors

Comment (crawfxrd) [reply src/board/system76/darp8/board.mk:26]: The killawatt is measuring ~16W with either value.

Comment (crawfxrd) [reply src/board/system76/darp8/board.mk:26]: Are we programming the oz26786 correctly?

The datasheet shows `AdapterCurrent()` uses each bit as a power-of-2 value to add to the current and not an immediate mA value.

Comment (crawfxrd) [reply src/board/system76/darp8/board.mk:26]: Okay, bit 0 is a 2mA weight. That's why it has to be divided by 2.

**Review by jackpot51 — APPROVED** [review]

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**